### PR TITLE
livecheck: modify Formula urls

### PR DESCRIPTION
**Pull request by nandahkrishna**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Changes required by `livecheck`'s Formula URL reference feature.

Changes:
* `ivykis`: Appending a `/` to the `homepage` url for `livecheck` to use it
* `flawfinder`: It seems like the repository is now at https://github.com/david-a-wheeler/flawfinder.git and not the previous `head` URL, which doesn't show anything (for me at least).
* `mdds`: Adding a `.git` suffix to the `homepage` URL as `livecheck` requires it
* `pass`: Adding a `.git` suffix to the `head` URL as `livecheck` requires it

### CODE REVIEW

File: Formula/mdds.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Mdds < Formula
   desc "Multi-dimensional data structure and indexing algorithm"
-  homepage "https://gitlab.com/mdds/mdds"
+  homepage "https://gitlab.com/mdds/mdds.git"
```
Comment (SMillerDev) [inline Formula/mdds.rb:3]: Shouldn't this be HEAD instead of homepage?

Comment (nandahkrishna) [reply Formula/mdds.rb:3]: Oh yeah, I'll change that.

**Comment by nandahkrishna** [comment]
Changes have been made and CI is 🟢.

**Review by SMillerDev — APPROVED** [review]
Thanks @nandahkrishna ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**